app/main/posts/views/filters/filter-posts.html

```diff
@@ -26,5 +26,5 @@
         </a>
     </div>
 
-    <filters-dropdown uib-dropdown-menu class="dropdown-menu" dropdown-status="status" filters-var="filters" ng-model="filters" cancel="cancel" apply-filters="applyFilters"></filters-dropdown>
+    <filters-dropdown view="view" uib-dropdown-menu class="dropdown-menu" dropdown-status="status" filters-var="filters" ng-model="filters" cancel="cancel" apply-filters="applyFilters"></filters-dropdown>
```

Comment:
I can't figure out how to access view name (if it's even possible) in the filters-dropdown so I send it from the filter-posts so I don't have to mess with parents and assume things